### DISCUSSION TITLE
Issue with large video files

### DISCUSSION THREAD

**Discussion by FilipLaurentiu** [haugene/docker-transmission-openvpn#2774] [Q&A] [answered]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

Not relevant

### Current Behavior

It's very weird and I did a lot of debugging for this issue. I use the container on my synology nas for different type of files including large video files (movies). I don't watch a lot of movies, but I usually download the movie and then use Plex from my nas to watch the movie on different devices.
Recently (months, idk if happened before) Plex had a lot of problems streaming the videos and I thought that it's a Plex problem, or any other problem, didn't thought about the torrent client. 
I took into consideration the performance of my NAS so I copy the file to my laptop..same problem, the video just crash at some random point every time. 
I run out of ideas but I had one last stupid idea, to download the same movie straight on my laptop, using normal Transmission, and try to see if  it's the same. Did that, no problem, same movie no problem. Then I do the same on the NAS, I used another torrent client to download a movie and it worked without any problem.
I don't know what happen.. the only 'weird' thing that I notice is with the metadata of the video file that Transmission-openvpn, in the properties of the file is written "Size of file: 20 GB, Size on disk: 7 GB". With the normal transmission client, size of file and size on disk is the same.
I know it's a weird issue .. hard to explain, maybe the files are not linked correctly

### Expected Behavior

No issues with large video files

### How have you tried to solve the problem?

1) use normal transmission

### Log output

Don't think it's relevant, no errors in the logs

### HW/SW Environment

```markdown
- OS: DSM 7.2.1-69057 Update 3
- Docker:  Server Version: 20.10.23 (I think it's a synology specific)
```

**Comment by pkishino** [comment]
have you tried using this container on your laptop instead of the Synology ?
this is more likely to do with your docker on dsm, your Nas etc..

**Reply by FilipLaurentiu** [reply]
Hmm, ok, I will, then come back with the results 🤝

**Reply by FilipLaurentiu** [reply]
I think I found the real problem. The `incomplete` folder was full of files that were not moved completely. 
If the container crash, the process of moving complete torrents is started again or is aborted ? I don't see any relevant error in the logs about this.

**Reply by pkishino** [reply, marked as answer]
Have you checked the transmission logs, not the container logs?
Please see if you find anything on the transmission repo issue tracker, this is not related to the container itself.

**Reply by FilipLaurentiu** [reply]
Ok, I understand. Thanks 🤝
